aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -971,6 +989,11 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   m.impl("is_complex", native::is_complex);
   m.impl("conj", native::conj);
 
+  // inplace operations
+  m.impl("fill_.Scalar", fill_inplace_scalar_batching_rule);
+  m.impl("fill_.Tensor", fill_inplace_tensor_batching_rule);
+  m.impl("zero_", zero_inplace_batching_rule);
```

Comment:
nit: we can just use `m.impl("zero_", native::zero_)` here to reuse some code

Reply:
@zou3519 Thanks so much for the hint of using `native::zero_` instead of writing batching rules for `zero_` explicitly.
However, `native::zero_` may cause the runtime exception when handling batched inputs.
Test code:
```python
x = torch.ones(3, 5)
zero_f = lambda x: x.zero_()
vmap(Tensor.zero_)(x)
``` 

would throw
```
Traceback (most recent call last):
  File ".\testcode\fill_vmap.py", line 13, in <module>
    vmap(torch.Tensor.fill_)(x, y)
  File "e:\repo\ml\github\pytorch\torch\_vmap_internals.py", line 250, in wrapped
    batched_outputs = func(*batched_inputs)
RuntimeError: batched INTERNAL ASSERT FAILED at "..\\aten\\src\\ATen\\VmapTransforms.cpp":45, please report a bug to PyTorch. logicalToPhysical(tensor) should only be passed a BatchedTensor
```

Reply:
Interesting. thanks for checking!